.github/workflows/master.yml

```diff
@@ -106,6 +106,7 @@ jobs:
     - name: Install dependencies
       env:
         INSTALL_LARGE_PYTHON_DEPS: true
+        INSTALL_SMALL_PYTHON_DEPS: true
```

Comment:
Why do we need to install smalls here as well?

Reply:
It's just the way the install-common-deps.sh helper script is written, we independently flag-on installation of small/large dependencies. In the future, maybe we should refactor the helper script so that you declaratively specify what job type you're running (small or large python tests, java tests, etc) and it installs the appropriate dependencies - this would make it easier to e.g. repro the steps of a github actions run on your local machine.